Source/GlideHQ/tc-1.1+/fxt1.c

```diff
@@ -755,7 +755,7 @@ fxt1_quantize_MIXED1 (dword *cc,
     /* left microtile */
     if (maxColL == -1) {
 	/* all transparent black */
-	cc[0] = ~0UL;
+	cc[0] = 0xFFFFFFFFUL;
```

Comment:
It's a warning fix.  C warnings are not generally about style.

As explained in the #1085 copy of this pull request:
```
~0UL on 32-bit:  0000 0000 FFFF FFFF
~0UL on LP64 :  FFFF FFFF FFFF FFFF
```

As `cc[0]` is of type `typedef unsigned int dword;`, ~0UL risks overflow.